packages/ember/index.js

```diff
@@ -106,6 +106,7 @@ import {
   TextArea,
   isSerializationFirstNode,
 } from 'ember-glimmer';
+// eslint-disable-next-line import/no-unresolved
 import VERSION from './version';
```

Comment:
We could always have a placeholder file for this and clobber it with the generated one

Reply:
Yep, I am planning to add a file for the new "public modules api" for it instead though. I just wanted to land this first, as it massively helps to have lint failures on invalid module imports (as opposed to running the tests and hoping that there are failures...).